from dm_api_account.models.general_error_model import GeneralError